src/rp2_common/pico_runtime_init/include/pico/runtime_init.h

```diff
@@ -80,7 +80,7 @@ void runtime_init_bootrom_reset(void);
 #endif
 
 // ---------------------------------------------------------------------------------------
-// Non-boot core eset of bootrom state, not needed if only using core 0 not used on RP2040
+// Non-boot core reset of bootrom state, not needed if only using core 0 not used on RP2040
```

Comment:
Should there be a comma after "core 0" here, to avoid this being misread as "core 0 not used on RP2040" ?